### ISSUE TITLE
KNeighboursClassifier converter issues

### ISSUE THREAD

**Issue by prabhat00155** [onnx/sklearn-onnx#203]
Probability score mismatches with the following combination of (algorithm, metric) hyper-parameters:
ball_tree, p | ball_tree, cityblock | ball_tree, l1 | ball_tree, chebyshev | ball_tree, infinity | ball_tree, hamming | ball_tree, canberra | ball_tree, braycurtis | ball_tree, matching | ball_tree, jaccard | ball_tree, dice | ball_tree, kulsinski | ball_tree, rogerstanimoto | ball_tree, russellrao | ball_tree, sokalmichener | ball_tree, sokalsneath | kd_tree, p | kd_tree, cityblock | kd_tree, l1 | kd_tree, chebyshev | kd_tree, infinity | brute, cityblock | brute, cosine | brute, manhattan | brute, braycurtis | brute, canberra | brute, chebyshev | brute, correlation | brute, cosine | brute, dice | brute, hamming | brute, jaccard | brute, kulsinski | brute, mahalanobis | brute, matching | brute, rogerstanimoto | brute, russellrao | brute, seuclidean | brute, sokalmichener | brute, sokalsneath | brute, sqeuclidean | brute, yule

**Comment by prabhat00155** [comment]
https://github.com/onnx/sklearn-onnx/pull/240
